packages/web/src/components/keyboard-shortcuts-dialog.tsx

```diff
@@ -36,16 +38,23 @@ export default function KeyboardShortcutsDialog({
               <ShortcutRow keys={['?']} description="show this dialog" />
               <ShortcutRow keys={['mod', ';']} description="open package.json" />
               <ShortcutRow keys={['mod', 'i']} description="open npm package install modal" />
```

Comment:
We probably shouldn't open this modal in readonly since it's for installing packages which cannot happen in read only.